style.css

```diff
@@ -11,238 +11,400 @@
 }
 
 body {
-  display: flex;
-  flex-direction: column;
-  max-width: 375px;
-  font-family: 'Poppins', sans-serif;
   background: #f7f7f9;
   margin: auto;
-  border: 2px solid #ccc;
 }
 
-.heading {
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
-  color: #172b4d;
-  text-transform: capitalize;
+.headr {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  background: #fff;
+  padding: 26px 7%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
 
-.btn {
-  margin-top: 1rem;
-  cursor: pointer;
-  text-transform: capitalize;
-  border: 1px solid #6070ff;
-  border-radius: 8px;
+.headr .logo {
   font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  text-align: center;
-  letter-spacing: 0.03em;
-  color: #396df2;
-  padding: 12px;
-  gap: 10px;
-  width: 124px;
-  background: #fff;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
 }
 
-/* header */
+.headr .logo .menuBtn {
+  display: none;
+}
 
-.navbar {
-  padding: 58px 24px 12px;
-  background-color: #fff;
+.headr .navbar {
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
 }
 
-.navbar .logo {
-  font-weight: 700;
-  font-size: 18px;
-  color: #6070ff;
+.headr .navbar .menuBtn {
+  display: none;
 }
 
-.navbar .menuBtn {
-  width: 18px;
-  height: 12px;
-  color: #6070ff;
+.headr .navbar a {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  color: #344563;
 }
 
-/* HEADLINE */
-
 .headline {
+  height: 846px;
+  justify-content: center;
+  gap: 12px;
+  background: #fff;
+  width: 100%;
   display: flex;
   flex-direction: column;
-  padding: 114px 24px;
-  gap: 12px;
   align-items: flex-start;
-  background: #fff;
-  background-image: url(unt/header-shapes\ mobile.png);
+  padding: 0 260px 0 260px;
+  padding-top: 287px;
+  padding-bottom: 243px;
+  background-image: url(unt/Header\ bg.png);
+  background-size: cover;
+  background-position: right;
   background-repeat: no-repeat;
 }
 
 .headline h1 {
-  font-style: normal;
+  font-size: 48px;
   font-weight: 700;
```

Comment:
- Great job for using the exact `font-weight: 700;`😁👍 however it is not applying because you didn't import the exact font, Please be sure to import the exact weight from [google font](https://fonts.google.com/) before using it 👍👌